tests/zkevm/test_worst_compute.py

```diff
@@ -237,23 +243,180 @@ def test_worst_modexp(
 
 
 @pytest.mark.valid_from("Cancun")
-def test_worst_ecrecover(
+@pytest.mark.parametrize(
+    "precompile_address,parameters",
+    [
+        pytest.param(
+            0x01,
+            [
+                # TODO: below inputs are covering a regular signature. Worth exploring
```

Comment:
For any kind of cryptography verification (signature verification, pairing check), the positive case is the worst case. If the input is invalid it means it has failed one of the verification steps.

Reply:
Cool, I'll remove that part of the comment then.